src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/CreateNSTreeModal/components/SampleIdInput/components/InputInstructions/strings.ts

```diff
@@ -0,0 +1,20 @@
+import { Pathogen } from "src/common/redux/types";
+import { PathogenConfigType } from "src/common/types/pathogenConfig";
+
+interface InputInstructionsStrings {
+  publicRepositoryIds: string;
```

Comment:
Minor nit on naming: I'd suggest changing this to `publicRepositoryIdType` -- I originally thought this referred to an actual list of IDs.